templates/repo/diff/box.tmpl

```diff
@@ -113,6 +113,11 @@
 									{{if $file.IsVendored}}
 										<span class="ui label gt-ml-3">{{$.locale.Tr "repo.diff.vendored"}}</span>
 									{{end}}
+									{{if and $file.Mode $file.OldMode}}
+										<span class="gt-ml-4 gt-mono">{{$file.OldMode}} &rarr; {{$file.Mode}}</span>
+									{{else if $file.Mode}}
+										<span class="gt-ml-4 gt-mono">0 &rarr; {{$file.Mode}}</span>
```

Comment:
How about just display the mode? Why we need that `0 ->`?

```suggestion
										<span class="gt-ml-4 gt-mono">{{$file.Mode}}</span>
```

Reply:
I made it similar to GitLab to provide better visual clarity that this is a new file.
Would you like me to delete it?

Reply:
We can not get to know whether it's a new file from a single mode?
Yes.

Reply:
done https://github.com/go-gitea/gitea/pull/24966/commits/03477d5dc75a5c311c673bddafce66fc88b71dfc

Reply:
Do we actually need to display the 10644 mode (bit is off)? Does GitHub do it? I think 10755 (bit is on) is the interesting one, and a un-executable file is not that interesting to display on every new file.

Reply:
I just tested on GitHub, they don't show mode on new files at all:

![image](https://github.com/go-gitea/gitea/assets/115237/f7d31702-ceae-481b-b8cb-82d03c242246)

I think we should change it that mode on new files is only shown when executable bit is set, or not show at all. Showing it on non-executable files is unnecessary noise imho.

Reply:
I believe it's better to show the permissions for newly created files.

At work, my colleagues sometimes ask me why a new script isn't running in CI. Some of them use Windows, and checking file permissions can be a challenge for them.
It's much easier for them to look at the commit on GitLab and see the permissions with which the file was created.
If we don't display this information, it will be more difficult to check the permissions on Gitea.

Showing the permissions selectively, in my opinion, is not very convenient because then people would need to know the default permissions since they are not displayed.

Reply:
It'd still be visible in single file view, but let's see it it goes then.